content/blog/2021/03/camel-quarkus-release-1.8.0/index.md

```diff
@@ -0,0 +1,107 @@
+[Groovy](https://camel.apache.org/camel-quarkus/latest/reference/extensions/groovy-dsl.html),
+[YAML](https://camel.apache.org/camel-quarkus/latest/reference/extensions/yaml-dsl.html),
+[Kotlin](https://camel.apache.org/camel-quarkus/latest/reference/extensions/kotlin-dsl.html)
+or [JavaScript](https://camel.apache.org/camel-quarkus/latest/reference/extensions/js-dsl.html)).
+With Camel Quarkus you still need the `pom.xml` file with the appropriate dependencies in addition to the route definition files.
+Except for XML and YAML, these DSLs are supported only in JVM mode, because they load and compile the routes at runtime.
+
+## New extensions
+
+Except for the new DSLs mentioned above, there are six new extensions:
+
+* [Azure Storage Data Lake](https://camel.apache.org/camel-quarkus/latest/reference/extensions/azure-storage-datalake.html) (JVM only)
+* [Freemarker](https://camel.apache.org/camel-quarkus/latest/reference/extensions/freemarker.html) (native)
+* [HL7](https://camel.apache.org/camel-quarkus/latest/reference/extensions/hl7.html) (native)
+* [Huawei Cloud SMN](https://camel.apache.org/camel-quarkus/latest/reference/extensions/huaweicloud-smn.html) (JVM only)
+* [Paho MQTT 5](https://camel.apache.org/camel-quarkus/latest/reference/extensions/paho-mqtt5.html) (native)
+* [Stitch](https://camel.apache.org/camel-quarkus/latest/reference/extensions/stitch.html) (JVM only)
+
+These three extensions are now supported in native mode:
+
+* [LRA](https://camel.apache.org/camel-quarkus/latest/reference/extensions/lra.html)
+* [Nitrite](https://camel.apache.org/camel-quarkus/latest/reference/extensions/nitrite.html)
+* [Splunk](https://camel.apache.org/camel-quarkus/latest/reference/extensions/splunk.html)
+
+## Deprecated extensions
+
+Deprecated extensions:
+
+* `camel-quarkus-componentdsl` and `camel-quarkus-endpointdsl` are a part of `camel-quarkus-core`
+  since long, so you can remove them safely making sure that your application depends on `camel-quarkus-core`
+  either directly or indirectly.
+* Similarly, `camel-quarkus-main` is now a part of `camel-quarkus-core` and you can remove it safely making
+  sure that your projects depends on `camel-quarkus-core` at least transitively.
+
+## Breaking changes:
+
+The following deprecated extensions were removed in this release:
+
+* WebSocket JSR 356 (replaced by [Vert.x WebSocket](https://camel.apache.org/camel-quarkus/latest/reference/extensions/vertx-websocket.html))
+* All AWS SDK v1 extensions (replaced by their AWS SDK v2 counterparts)
+
+A part of the functionality originally present in `camel-quarkus-xml-io` has moved to
+`camel-quarkus-xml-io-dsl`. Hence when you see e.g. `Cannot find RoutesBuilderLoader in classpath supporting file extension: xml` in your log, you'll need to replace `camel-quarkus-xml-io` with `camel-quarkus-xml-io-dsl`.
+
+
+## Full Changelog of Camel Quarkus 1.8.0
+
+* [Fixed issues](https://github.com/apache/camel-quarkus/milestone/12?closed=1)
```

Comment:
You could point to a release also, something like:

```suggestion
* [Fixed issues](https://camel.apache.org/releases/q-1.8.0/)
```

That would require creating a `release-1.8.9.md` file here: https://github.com/apache/camel-website/tree/master/content/releases/q